commercetools-models/src/test/java/io/sphere/sdk/orders/commands/OrderImportCommandIntegrationTest.java

```diff
@@ -19,6 +19,8 @@
 import io.sphere.sdk.test.JsonNodeReferenceResolver;
 import io.sphere.sdk.test.SphereTestUtils;
 import io.sphere.sdk.types.CustomFields;
+import io.sphere.sdk.types.CustomFieldsDraft;
```

Comment:
Looks like these changes aren#t necessary. How about reverting them because they just add noise to our git history?